packages/dashboard/src/constants/wallet-labels.ts

```diff
@@ -11,3 +11,15 @@ export const getWalletLabels = () => ({
     TRANSFER: localize('Transfer'),
     WITHDRAWAL: localize('Withdrawal'),
 });
+
+interface IWalletLabels {
```

Comment:
(question) Don't we need to export the interface as well? No other usage?

Reply:
It doesn't have any other usage so far. I think we can export it later in case if needed.